protocol/xmrtaker/instance.go

```diff
@@ -173,6 +137,47 @@ func (inst *Instance) createOngoingSwap(s *swap.Info) error {
 	return nil
 }
 
+func (inst *Instance) completeSwap(s *swap.Info, skB *mcrypto.PrivateSpendKey) error {
```

Comment:
It's not obvious from the name how this function is used, so a comment might be helpful. As I understand it, this function only completes recovered swaps from a previous run of `swapd`. It looks like it will use the current value of "transfer back" (verses whatever value was set when the swap was started). It will also only only recover to the primary wallet address, not whatever address was used when the swap was started.  Both of those features are painful to support in the corner cases. Assuming we continue support, we should shine as much light into their darker corners as possible.